proposals/jvm-static-annotation-in-interface-companion.md

```diff
@@ -0,0 +1,36 @@
+# @JvmStatic for interface companion members
+
+* **Type**: Design proposal
+* **Author**: Mike Bogdanov
+* **Status**: Implemented in Kotlin 1.3
+* **Discussion**: [KEEP-150](https://github.com/Kotlin/KEEP/issues/150)
+
+## Summary
+
+Allow to use '@JvmStatic' annotation on interface companion members similar to class companion ones.
+
+## Description
+
+'@JvmStatic' annotation on interface companion members has same effect and similar restrictions as annotation on class companion members:
+* generates static method in interface that delegates to companion member
```

Comment:
Consider rephrasing like "generates static method in the interface. *This* static method will delegate to the companion member". Currently it reads like it's the "interface" who "delegates to companion member".